modules/include/cas_ioctl_codes.h

```diff
@@ -557,6 +557,9 @@ enum kcas_error {
 	/** Removing core failed and rollback failed too */
 	KCAS_ERR_DETACHED,
 
+	/** Cache is already in standby detached state */
+	OCF_ERR_STANDBY_DETACHED,
```

Comment:
This is, clearly, a KCAS error code as it originates from adapter, not from OCF : `OCF_` -> `KCAS_`

Reply:
Yes, you are right. I removed it.